include/boost/geometry/geometries/register/PolyhedralSurface.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef BOOST_GEOMETRY_GEOMETRIES_REGISTER_POLYHEDRALSURFACE_HPP
+#define BOOST_GEOMETRY_GEOMETRIES_REGISTER_POLYHEDRALSURFACE_HPP
+
+
+#include <boost/geometry/core/tag.hpp>
+#include <boost/geometry/core/tags.hpp>
+
+#define BOOST_GEOMETRY_REGISTER_POLYHEDRALSURFACE(PolyhedralSurface) \
+namespace boost { namespace geometry { namespace traits {  \
+    template<> struct tag<PolyhedralSurface> { typedef PolyhedralSurface_tag type; }; \
+}}}
```

Comment:
As I mentioned earlier, we can omit this (this whole file) in this PR. It's too early.
I'm curious if users will ever need it.

Reply:
Ok, should I delete the file or comment out the codes?